CS342MemberViews/Sources/DannyPark/DannyPark.swift

```diff
@@ -0,0 +1,68 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import InstructorViews
+import SwiftUI
+
+
+public struct DannyPark: View {
+    @Binding var navigationPath: NavigationPath
+    
+    
+    public var body: some View {
+        ScrollView(.vertical) {
+            VStack(spacing: 42) {
+                MapView()
+                    .frame(height: 300)
+                    .offset(y:-50)
```

Comment:
SwiftLint wants there to be a space between the colon and the value, like this:

```suggestion
                    .offset(y: -50)
```